src/modules/ticket/actions.ts

```diff
@@ -0,0 +1,18 @@
+import {ThunkAction} from 'redux-thunk';
+import {RootAction, RootState} from '@store/rootReducer';
+import {buyTicketCall} from 'src/api/api';
```

Comment:
I noticed the format '@' (instead of 'src/') is used everywhere in the code except for a couple places, so maybe it'd make sense to use it here as well? To keep the consistency of the imports. 🤔 
```suggestion
import {buyTicketCall} from '@api/api';
```